plugins/rapid7_insightidr/bin/komand_rapid7_insightidr

```diff
@@ -6,7 +6,7 @@ from sys import argv
 
 Name = "Rapid7 InsightIDR"
 Vendor = "rapid7"
-Version = "5.1.1"
```

Comment:
given that 5.1.2 has been merged to master in here https://github.com/rapid7/insightconnect-plugins/pull/2044/files was this pr based on an old version of develop ?

Reply:
there has been some conflicts .. should be 6.0.0